### ISSUE TITLE
distributive checks broken when upgrading from 1.03 to master (1.1)

### ISSUE THREAD

**Issue by ryane** [CiscoCloud/mantl#1288]
- Ansible version (`ansible --version`): 1.9.4
- Python version (`python --version`): 2.7.6
- Git commit hash or branch: master
- Cloud Environment: aws
- Terraform version (`terraform version`): v0.6.11

see #1268 

I tried upgrading a 1.0.3 cluster with #1268  and I still end up with the distributive files in the consul directory and the distributive directory:

```
$ sudo ls -al /etc/consul
total 72
drwxrwx---.  3 consul consul 4096 Mar 17 20:32 .
drwxr-xr-x. 95 root   root   8192 Mar 17 20:22 ..
-rw-r--r--.  1 consul consul  163 Mar 17 19:57 acl.json
-rw-r--r--.  1 root   root     57 Mar 17 19:58 agent_acl.json
-rw-rw-r--.  1 consul consul  753 Mar 17 20:21 consul.json
-rw-r--r--.  1 root   root    995 Mar 17 19:59 consul.nginx
-rw-r--r--.  1 root   root     41 Jan 19 14:51 consul-ui.json
-rw-r--r--.  1 consul consul  276 Mar 17 19:52 distributive-common-check.json
-rw-r--r--.  1 root   root    296 Mar 17 20:32 distributive-consul-check.json
-rw-r--r--.  1 root   root    299 Mar 17 20:32 distributive-dnsmasq-check.json
-rw-r--r--.  1 root   root    302 Mar 17 20:32 distributive-logstash-check.json
-rw-r--r--.  1 root   root    299 Mar 17 20:32 distributive-mantlui-check.json
-rw-r--r--.  1 root   root    333 Mar 17 20:32 distributive-zookeeper-check.json
-rw-r--r--.  1 consul consul  176 Mar 17 19:57 docker-service.json
-rw-r--r--.  1 root   root    312 Mar 17 20:02 marathon.json
drwxrwxr-x.  2 consul consul   55 Mar 17 19:57 ssl
-rw-r--r--.  1 root   root    261 Mar 17 20:00 zk-consul.json
```

```
ls -al /etc/distributive.d/
total 36
drwxr-xr-x.  2 root root 4096 Mar 17 20:24 .
drwxr-xr-x. 95 root root 8192 Mar 17 20:22 ..
lrwxrwxrwx.  1 root root   35 Mar 17 19:52 common.json -> /usr/share/distributive/common.json
-rw-rw-r--.  1 root root 1566 Feb  9 21:54 distributive-consul.json
-rw-rw-r--.  1 root root 1336 Mar 14 22:01 distributive-dnsmasq.json
-rw-rw-r--.  1 root root 1178 Mar 11 14:04 distributive-logstash.json
-rw-rw-r--.  1 root root  910 Mar 11 19:28 distributive-mantlui.json
-rw-rw-r--.  1 root root 1213 Feb  9 21:57 distributive-zookeeper.json
```

There were no errors during the ansible run. And all nodes are unhealthy b/c of this after the upgrade.

**Comment by siddharthist** [comment]
@ryane Thanks for all your testing and feedback, I think I've got a solution on the way. I'm pretty sure that the common role is the problematic one.

**Comment by ryane** [comment]
fixed in #1296
